project/Dependencies.scala

```diff
@@ -3,6 +3,8 @@ import sbt.Keys._
 
 object Dependencies {
 
+  lazy val grpc = "io.chrisdavenport" %% "http4s-grpc-google-cloud-pubsub-v1" % "1.108.0+0.0.6"
```

Comment:
Wondering what kind of impact on bincompat this will have, Chris states the libraries don't guarantee it due to the underlying protos potentially changing. I wonder if we should generate the sources ourselves so mima can check them and we can change versions based on that?

Reply:
That's a good idea, let me investigate it.